en/admin/config.rst

```diff
@@ -1777,6 +1782,38 @@ The following are parameters related to the query plan cache functionality. The
 
     **max_filter_pred_cache_entries** is a parameter used to specify the maximum number of filtered index expressions. The filtered index expressions are stored with them complied and can be immediately used in server. If it is not stored in cache, the process is required which filtered index expressions are fetched from database schema and interpreted.
 
+.. _query-cache-parameters:
+
+Query Cache-Related Parameters
+-----------------------------------
+
+The following are parameters related to the query cache functionality. The type and value range for each parameter are as follows:
+
++-------------------------------+--------+----------+----------+----------+
+| Parameter Name                | Type   | Default  | Min      | Max      |
++===============================+========+==========+==========+==========+
++-------------------------------+--------+----------+----------+----------+
+| max_query_cache_entries       | int    | 200      | 1        | INT_MAX  |
++-------------------------------+--------+----------+----------+----------+
+| max_query_cache_pages         | int    | 1,000    | 1        | INT_MAX  |
++-------------------------------+--------+----------+----------+----------+
+
+**max_query_cache_entries**
+
+    **max_query_cache_entries** is a parameter to configure the maximum number of query to be cached. If the **max_query_cache_entries** parameter is configured to 0 or a negative value, paramter configuration error occurs; if it is configured to an integer value equal to or greater than 1, a specified number of queries are cached with the result.
```

Comment:
I recommend that user can be set max_query_cache_entries to 0 to change query cache disabled.

Reply:
Code should also be modified as you comment.
I will first modify the related code.